### PR TITLE
[WIP] Move to 2019

### LINKED ISSUE #24: Put the CFP review board on the main event page
Right now we have the CFP review board in a separate area on the site so that it links from the drop down of CFP. As a result of this, moving over to the 2019 CFP review board we are not going to have it linked any longer. We should include the CFP Review board on the `2018.md` and then `2019.md` pages to preserve history.

### PR DESCRIPTION
**Pull request by sparklespdx**
Resolves https://github.com/BSidesPDX/bsidespdx.org/issues/24 and most of https://github.com/BSidesPDX/bsidespdx.org/issues/19

Things we still need are:

- Updated CFP google form
- Updated venue information
- Updated registration link / Dates for when registration will open

### CODE REVIEW

**Comment by sparklespdx** [comment]
I don't recommend merging this until we have the above 3 items sorted

**Comment by sparklespdx** [comment]
I decided to leave the old content as comments to make it easier on myself when putting in new content, we can remove that if others think it is messy

File: events/2019/review-board.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+layout: page
+title: "BSidesPDX CFP Review Board"
+permalink: /events/2019/review-board.html
+---
+
+As BSidesPDX has evolved over the years into a larger event, the process of curating the content has evolved as well. In an effort to recognize that the goal of BSidesPDX is to support the local information security community, we've recruited a set of local experts to help with the selection process.
+
+We are still in the process of recruiting a CFP review board for BSides PDX 2019. Stay tuned!
+
+If you are interested in participating on the CFP board, please <a href="/about/contact.html">contact us!</a>
```
Comment (tophertimzen) [inline events/2019/review-board.md:11]: The CFP review board is an invite only sort of deal, similar to keynotes for the conference, I would remove this grammar on line 11.

File: events/cfp.md [hunk 2]
```diff
@@ -3,7 +3,8 @@ layout: page
 title: "BSidesPDX Call For Papers"
 permalink: /cfp.html
 ---
+The BSides PDX 2019 Call For Papers is coming soon! Join our [mailing list](/about/contact.html) for announcements.
```
Comment (tophertimzen) [inline events/cfp.md]: We can leave the below CFP google form embedded, the CFP is always open and we'll be announcing it shortly.

**Comment by tophertimzen** [comment]
Having the comments on all of the 2019 pages makes it look a little bit messy when we can copy specific formatting from 2018 markdown pages. I would vote we remove them just to make the pages look a bit more tidy when opened in a text editor.

**Comment by sparklespdx** [comment]
-  Venue page updated
- CFP form left as it was last year, verbage needs to be updated but I leave that to @tophertimzen 
- Sounds like registration will be updated when it is open

I think this is OK to merge.

**Review by tophertimzen — COMMENTED** [review]
Merged!

**Pull request merged**